src/rust-crypto/rust-crypto.ts

```diff
@@ -1616,6 +1615,13 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, CryptoEventH
         });
     }
 
+    /**
+     * Implementation of {@link CryptoApi#markAllTrackedUsersAsDirty}.
+     */
```

Comment:
note that `RustCrypto` has a "CryptoApi implementation" section, which isn't here. Though, as previously noted, I don't think this actually belongs in `CryptoApi`.

Reply:
yep, it's gone.